alica_engine/src/engine/parser/ModelFactory.cpp

```diff
@@ -1406,24 +1406,24 @@ void ModelFactory::removeRedundancy()
 
 map<int64_t, AlicaElement*>* ModelFactory::getElements()
```

Comment:
brr. This whole (non-const!) accessor should go away. 
It currently is only used in alica_engine/src/engine/collections/RobotEngineData.cpp:68 inside an assert to verify uniqueness of generated ids. That could be replaced with a `bool isUnique(id) const` or even `int64_t makeUnique()`.